### PR TITLE
fix(ui-color-picker): make ColorPicker tooltip VoiceOver focusable

### PR DESCRIPTION
**Pull request by balzss**
Closes: INSTUI-4259

test plan:

- check colorpicker example with tooltip if voiceover reads the tooltip context correctly

### CODE REVIEW

**Review by ToMESSKa — APPROVED** [review]

**Review by matyasf — CHANGES_REQUESTED** [review]
I dont like that the SR is focusing on nothing and the tooltip does not show with this solution.

I would solve the underlying issue, that icons set `aria-hidden` to `true` (in `InlineSVG`) by default. You just need a small change there to allow this attribute:

image

**Comment by balzss** [comment]
@matyasf I've went with a different approach. According to [w3c](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/) and [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) tooltips should be focusable with keyboard.

**Review by matyasf — CHANGES_REQUESTED** [review]
I dont think this is a good solution. NVDA does not read the tooltip. Also its wonky on VO too, when I move the SR focus it does not read it, but after I've tabbed trough it, it will read it
I think its because the `svg` icon is set to `aria-hidden` which is a no for focusable elements https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#description , so NVDA just removes is from the SR DOM

**Comment by balzss** [comment]
@matyasf can you please check it again? now it also reads the tooltip when the input is focused but I don't think we can prevent that. I think it always reads the label when an input gets focused...

also fyi: it has a ts error that's probably introduced from [this pr](https://github.com/instructure/instructure-ui/pull/1771/files#diff-aacd5a00cfdc942924434e62ae19741c1a8d335477045d8cb9cdc143745454dfR397) so a [temporary ts-ignore](https://github.com/instructure/instructure-ui/pull/1771/files#diff-aacd5a00cfdc942924434e62ae19741c1a8d335477045d8cb9cdc143745454dfR397) was added

File: packages/ui-svg-images/src/InlineSVG/index.tsx [hunk 1]
```diff
@@ -150,7 +150,6 @@ class InlineSVG extends Component<InlineSVGProps> {
       title,
       description,
       focusable,
-      children,
```
Comment (matyasf) [inline packages/ui-svg-images/src/InlineSVG/index.tsx:153]: why did you remove this?

**Review by matyasf — APPROVED** [review]
nice work!

File: packages/ui-color-picker/src/ColorPicker/index.tsx [hunk 2]
```diff
@@ -389,9 +389,22 @@ class ColorPicker extends Component<ColorPickerProps, ColorPickerState> {
       <span>
         <span css={styles?.label}>{label}</span>
         <span>
-          <Tooltip renderTip={tooltip}>
-            <IconInfoLine tabIndex={-1} />
-          </Tooltip>
+          <InstUISettingsProvider
+            theme={{
+              themeOverrides: {
+                componentOverrides: {
```
Comment (matyasf) [inline packages/ui-color-picker/src/ColorPicker/index.tsx:395]: I did not knew that this is a valid syntax... easier one would be

```
  theme={{
    componentOverrides: {
      BaseButton: {
        smallHeight: 'auto'
      }
    }
  }}
```

**Pull request merged**